deepwell/src/endpoints/locale.rs

```
@@ -93,7 +93,7 @@ pub async fn translate_strings(
             ctx.localization()
                 .translate(&locales, &message_key, &arguments)?;
 
-        output.insert(message_key, translation.to_string());
+        output.insert(message_key, Some(translation.to_string()));
```

What does null here represent? If a translation key cannot be found, it returns an error, not `None`. Do you think we should change this behavior?

---

This was left over from when the fallback was not implemented and I just made my own endpoint that for looped the locales. However in the current fallback version, the key not found error obscures all the other existing keys that are requested in the same call.